rachellegelden/Home-Together-Canada
# HTC-566: Member Listing Cards

…s for tooltip

# [566](https://github.com/rachellegelden/Home-Together-Canada/issues/566)

## Summary
Added memberListingCard component with snapshot test
Added controls
Updated parent tests to pass tooltip snapshots

## Relevant Motivation & Context
Needed for search listing feature

## Testing Instructions
npm test in root
run locally and view functionality in /services
check console for errors

## Developer checklist prior to opening this pull request:


## Reviewer
